src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -511,6 +511,8 @@ void GeneralPage::initLayout(SettingsLayout &layout)
     layout.addCheckbox("Show parted users (< 1000 chatters)", s.showParts);
     layout.addCheckbox("Lowercase domains (anti-phishing)", s.lowercaseDomains);
     layout.addCheckbox("Bold @usernames", s.boldUsernames);
+    layout.addCheckbox("Mention users without prefix or suffix",
```

Comment:
This should probably be something along the lines of "Try to find usernames without @ prefix"

The variable name should also somewhat match that

Reply:
Updated in 44263e7